client/webpack-utils/extract-manifest-plugin.js

```diff
@@ -0,0 +1,81 @@
+const PLUGIN_NAME = 'ExtractManifestPlugin';
+
+class ExtractManifestPlugin {
+	constructor( options = {} ) {
+		this.options = {
+			jsExtension: '.min.js',
+			cssExtension: '.min.css',
+			manifestFilename: 'manifest.js',
```

Comment:
Can we also generate a chunk hash for the `manifest` chunk and add it to the filename?

If the filename is just `manifest.js`, the only good way how to ship it to the browser is to inline it in a `script` tag, because then the filename doesn't matter.

If we want to be flexible, then it should be also possible to load it as URL:
```
<script src="/calypso/evergreen/manifest.bb6fb226f19be0fdc380.min.js"></script>
```

Reply:
Make sense, let me check if I can get webpack to hash it as well.